docs/diagrams/ModelClassDiagram.puml

```diff
@@ -16,9 +16,23 @@ Class UniquePersonList
 Class Person
 Class Address
 Class Email
-Class Name
```

Comment:
wah, i feel like this is abit too complicated, should we omit some lower-level details like the components of event/person/logs?